apis/swagger.yml

```diff
@@ -1511,6 +1511,27 @@ definitions:
           This dfget is treated a client and carries a client ID.
           Thus, multiple dfget processes on the same peer have different CIDs.
 
+  HeartBeatResponse:
```

Comment:
Where is the `HeartBeatResponse` be used in swagger?

Reply:
API /peer/heartbeat.

Reply:
Yeah, I find this api, but its response definition is not `HeartBeatResponse`.

Reply:
The definition in file api.md?

Reply:
If so, i'll make it complete soon.